pages/vi/vi-bellapps.md

```diff
@@ -3,65 +3,76 @@
 ## Objectives
 
 * Understand how the BeLL-Apps interface is structured
```

Comment:
How's about we use these phase instead? "Understand how Bell-Apps interface is structured"?